drivers/hud_twenty_twenty_four_to_twenty_twenty_six/spec/models/csv_transformer_spec.rb

```diff
@@ -28,6 +28,43 @@
     expect(compare_test_results).to eq([])
   end
 
+  it 'creates CustomGender.csv with gender data from Client.csv' do
+    skip('Ignoring test, no input files') unless File.exist?(File.join(SOURCE_DIR, 'Export.csv'))
+
+    HudTwentyTwentyFourToTwentyTwentySix::CsvTransformer.up(SOURCE_DIR, DEST_DIR)
+
+    custom_gender_file = File.join(DEST_DIR, 'CustomGender.csv')
+    expect(File.exist?(custom_gender_file)).to be true
+
+    # Verify the file has content (header + data rows)
+    csv_content = CSV.read(custom_gender_file, headers: true)
+    expect(csv_content.length).to be > 0
+
+    # Verify required columns exist
+    headers = csv_content.headers
+    expect(headers).to include('PersonalID', 'Woman', 'Man', 'NonBinary', 'DateCreated', 'DateUpdated')
```

Comment:
I wonder if we can pull this expectation from `drivers/hmis_csv_twenty_twenty_six/app/models/hmis_csv_twenty_twenty_six/loader/custom/custom_gender.rb`
```
HmisCsvTwentyTwentySix::Loader::Custom::CustomGender.hud_csv_headers`